src/Compilers/Core/Portable/StrongName/StrongNameKeys.cs

```diff
@@ -88,14 +88,13 @@ internal static StrongNameKeys Create(ImmutableArray<byte> publicKey, CommonMess
 
         internal static StrongNameKeys Create(string keyFilePath, CommonMessageProvider messageProvider)
         {
-            if (string.IsNullOrEmpty(keyFilePath))
+            if (string.IsNullOrEmpty(keyFilePath) || !PathUtilities.IsAbsolute(keyFilePath))
```

Comment:
I'd say that the caller should call this method with a relative path and it can assert that options have errors in this case. Unfortunately we cannot assert presence of errors here.